player/lua/stats.lua

```diff
@@ -0,0 +1,734 @@
+-- Display some stats.
+--
+-- Please consult the readme for information about usage and configuration:
+-- https://github.com/Argon-/mpv-stats
+--
+-- Please note: not every property is always available and therefore not always
+-- visible.
+
+local mp = require 'mp'
+local options = require 'mp.options'
+
+-- Options
+local o = {
+    -- Default key bindings
+    key_oneshot = "i",
+    key_toggle = "I",
+    key_page_1 = "1",
+    key_page_2 = "2",
+    key_page_3 = "3",
+
+    duration = 4,
+    redraw_delay = 1,                -- acts as duration in the toggling case
+    ass_formatting = true,
+    persistent_overlay = false,      -- whether the stats can be overwritten by other output
+    print_perfdata_passes = false,   -- when true, print the full information about all passes
+    filter_params_max_length = 100,  -- a filter list longer than this many characters will be shown one filter per line instead
+    debug = false,
+
+    -- Graph options and style
+    plot_perfdata = true,
+    plot_vsync_ratio = true,
+    plot_vsync_jitter = true,
+    skip_frames = 5,
+    global_max = true,
+    flush_graph_data = true,         -- clear data buffers when toggling
+    plot_bg_border_color = "0000FF",
+    plot_bg_color = "262626",
+    plot_color = "FFFFFF",
+
+    -- Text style
+    font = "Source Sans Pro",
+    font_mono = "Source Sans Pro",   -- monospaced digits are sufficient
```

Comment:
What happens if the user doesn't have this font installed?

Reply:
Fallback as performed by libass or whatever it uses to lookup fonts.

Reply:
Is there some generic way to ask any monospace font?

On a system which doesn't have this font (and arguably most systems don't have it), it'll probably choose a non monospace one, which will look ugly, even if the user does have other mono fonts installed, wouldn't it?

Reply:
>Is there some generic way to ask any monospace font?

Not for a lua script.